modules/ROOT/pages/custom-view.adoc

```diff
@@ -0,0 +1,64 @@
+= Custom View
+:navtitle: Custom View
```

Comment:
@FarzadHayat Should this be "Custom view" just to be in sync with similar pages.
Eg: http://docs-feature-70-doc-1921.staging.tiny.cloud/docs/tinymce/latest/contextform/

Reply:
```suggestion
:navtitle: Custom view
```